benchmark/process_single_doc.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+import time
+from pathlib import Path
+
+import numpy as np
+from megaparse import MegaParse
+from megaparse.parser.unstructured_parser import UnstructuredParser
+from megaparse_sdk.schema.parser_config import StrategyEnum
+
+N_TRY = 1
+
+
+async def process_file(megaparse: MegaParse, file_path: str | Path):
```

Comment:
Question : Is it overkill to do timer decorator in this case ? That we could reuse for other processing